lib/internal/Magento/Framework/App/FeedImporterInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\App;
+
+interface FeedImporterInterface
+{
+
+    /**
+     * @throws \Magento\Framework\Exception\FeedImporterException
+     * @param  array  $data
+     * @param  string $format
+     * @return FeedInterface
+     */
+    public function importArray(array $data, $format = 'atom');
```

Comment:
I see your point but I did it because I wanted to replicate the original importArray method from Zend_Feed which has those arguments. What would be the best thing to do in this case?